charts/dso-ansible-job/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: cpn-ansible-job
 description: Creates Job for Socle services post-configuration.
 type: application
-version: 1.1.0
+version: 1.1.1
```

```suggestion
version: 1.2.0
```
Par rapport au semver non ?